the-9-exercises/src/components/Fetch.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import UserDisplayComponent from "./UserDisplayComponent";
+
+export default function Fetch() {
+  const [user, setUser] = useState([]);
+  const [random, setRandom] = useState(0);
```

Comment:
As I can see you are using this state to detect that the button is clicked so that you can fetch new users. You actually don't need to have an extra state for this, you may put the code that fetch new users in a function and call that function when the button is clicked instead

Reply:
Did you manage to see this?

Reply:
> As I can see you are using this state to detect that the button is clicked so that you can fetch new users. You actually don't need to have an extra state for this, you may put the code that fetch new users in a function and call that function when the button is clicked instead

Did you manage to see this??